gui/create-item.lua

```diff
@@ -255,15 +256,16 @@ function createCorpsePiece(creator, bodypart, partlayer, creatureID, casteID, ge
   item_type = "GLOB"
  elseif CORPSE_PIECES[layerName] or HAIR_PIECES[layerName] then -- check if hair
   item_type = "CORPSEPIECE"
+ elseif LIQUID_PIECES[layerName] then -- check if hair
+  item_type = "GLOB" -- prolly need to do more work on this (it doesn't like spawning LIQUID items), but it works for now
  end
  local itemType = dfhack.items.findType(item_type..":NONE")
  local itemSubtype = dfhack.items.findSubtype(item_type..":NONE")
  local material = "CREATURE_MAT:"..raceName..":"..layerMat
  local materialInfo = dfhack.matinfo.find(material)
  local item_id = dfhack.items.createItem(itemType, itemSubtype, materialInfo['type'], materialInfo.index, creator)
  local item = df.item.find(item_id)
- item:setQuality(quality)
- local isCorpsePiece = false
+ -- item:setQuality(quality)
```

Comment:
is this because corpse pieces don't have a quality? it would be nice if that prompt didn't come up when it wasn't relevant.

Reply:
nothing happens if you do set it (i just thought passing it through would be funny), but ye it would.